models/Profile.js

```diff
@@ -99,5 +99,6 @@ const ProfileSchema = new Schema(
   { timestamps: true }
 );
 
+
```

Comment:
Removing extra line as Prettier will remove it on someone else's PR in the future - please check you have Prettier plugin installed on VScode, so this happens automatically when you save

```suggestion
```